package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
Note for maintainers: These changes are fine, as the current Devicon version is 2.15.1, not 2.15.0.